jdaviz/configs/imviz/plugins/parsers.py

```diff
@@ -104,11 +130,30 @@ def _parse_image(app, file_obj, data_label, show_in_viewer, ext=None):
         raise NotImplementedError(f'Imviz does not support {file_obj}')
 
     for data, data_label in data_iter:
+
+        # avoid duplicate data labels in colection
+        if data_label in app.data_collection.labels:
+            data_label = data_label + "_2"  # 0th-order solution as proposed in issue #600
+
         app.add_data(data, data_label)
         if show_in_viewer:
             app.add_data_to_viewer("viewer-1", data_label)
 
 
+def _info_nextensions(app, file_obj):
+    if _count_image2d_extensions(file_obj) > 1:
+        info_msg = SnackbarMessage(INFO_MSG, color="info", timeout=15000, sender=app)
```

Comment:
What is the unit of `timeout`? Hopefully not seconds because that is like over 4 hours.

Reply:
millisec